task1.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+a = np.random.random((1,500))
+
+x = np.arange(0.1,100,.1)
+y = (1/0.001)* np.log(x)
```

Comment:
You were supposed to plot the y values for a, rather than taking x. This is fine too!